planetaryimage/image.py

```diff
@@ -139,7 +139,12 @@ def start_byte(self):
     def shape(self):
         """Tuple of images bands, lines and samples."""
         try:
-            return self.data.shape
+            if len(self.data.shape) == 3:
```

Comment:
This commit modified `shape()` such that it always returns shape in the tuple of 3.

` a= PDS3Image.open('pds3_1band.IMG')`
` a.shape`
`  (1, 10, 10)`

`b = a.data[0, 3:8, 3:8]`
`b.shape`
`  (1, 5, 5)`  which was `(5,5)` before.